### PR TITLE
Update 3.9

### PR DESCRIPTION
**Pull request by Arishamays1**
## ☆Cloudpickle 3.0.0 Update ☆
[Jira Ticket](https://anaconda.atlassian.net/browse/PKG-5238)
[Upstream](https://github.com/cloudpipe/cloudpickle)
# Changes
- Updated version number and `sha256`
- To properly reference licensing files, the PyPI source URL has been updated  to `cloudpickle’s` upstream github url
- Added new dependency to `host`

### CODE REVIEW

**Review by ryanskeith — APPROVED** [review]
This looks ok to me.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,25 +1,24 @@
...
-  sha256: d89684b8de9e34a2a43b3460fbca07d09d6e25ce858df4d5a44240403b6178f5
+  url: https://github.com/cloudpipe/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 49cc8581ce305630b1e179bc083069b21b45db4731e5e3f2ea6fc40b131e1c55
 
 build:
   number: 0
-  skip: true # [py<36]
-  script: {{ PYTHON }} -m pip install . --no-deps -vv --no-build-isolation
+  skip: true # [py<38]
+  script:  {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
```
Comment (SC426) [inline recipe/meta.yaml:15]: I don't think `--ignore-installed --no-cache-dir -vv` is necessary anymore but I suppose it doesn't hurt either.

**Review by SC426 — APPROVED** [review]
LGTM!

File: recipe/meta.yaml (hunk 1 above)
Comment (JeanChristopheMorinPerso) [reply recipe/meta.yaml:15]: At least one `-v` is needed for us to view what's going on. `-vv` is excessive`.

Comment (Arishamays1) [reply recipe/meta.yaml:15]: Thanks guys! I'll make a quick little change!

**Pull request merged**